lib/cli/handlers/create.js

```diff
@@ -43,11 +43,11 @@ function _create({ fileMode = false, __fileDirname = '', awsProfile, sshPublicKe
     }
 
     if (!_.isEmpty(sslCertificatePath)) {
-        sslCertificatePath = resolvePath(sslCertificatePath);
+        sslCertificatePath = resolvePath(sslCertificatePath, __fileDirname);
```

Comment:
remove, we dont know what it's here for

Reply:
What do you mean "remove"? I use it, it's part of SSL provisioning process: https://github.com/orbs-network/nebula/blob/master/SSL.md